SanityPOC/ContentView.swift

```diff
@@ -43,7 +42,7 @@ struct ContentView: View {
 }
 
 struct TakeoverSheet: View {
-    @Binding var takeover: Takeover?
+    @State var takeover: Takeover?
```

Comment:
We could use `@ObservedObject` here if we wanted to pass in the whole viewModel object. Since we only need access to the `viewModel.takeover` its better to isolate just what we need and use the `@State` here instead.

Reply:
Another note is `@Binding` is generally used when we need to change a value from the view. ie a slider incrementing a value or something to that effect. Since we are not mutating the value of the takeover in this view there is no need for a binding, and instead we should use `@State`

Reply:
Oooooo this is GREAT callout!! That totally makes sense on State vs Binding.